docs/azure/aksextensions.md

```diff
@@ -0,0 +1,127 @@
+
+![AKS Diagnostics Webview](images/aksextensions/aks-diagnostics-webview.png)
+
+### AKS Periscope
+
+Right click on your AKS cluster and click on **Run AKS Periscope** to collect node and pod logs for your AKS cluster and to export them to an Azure storage account. Upon selecting the option, a web view will load providing you the option to generate a downloadable link for the collected logs as well as a shareable link with 7-day expiry.
+
+> If you are not seeing the logs for all the nodes, it is possible the logs were still uploading. Try clicking the **Generate Link** button again to load more logs.
+
+For more information, visit [AKS Periscope](https://github.com/Azure/aks-periscope).
+
+![AKS Periscope Webview](images/aksextensions/aks-periscope-webview.png)
+
+#### Configuring Storage Account
+
+Running the AKS Periscope requires you to have a storage account associated with the Diagnostic settings of your AKS cluster. If you have only one storage account associated with the Diagnostic settings of your AKS cluster, the collected logs will be stored in the associated storage account by default. If you have more than one storage account associated with the Diagnostics settings of your AKS cluster, then the extension will prompt you to choose the storage account for saving collected logs. If you don't have a storage account configured in the Diagnostic settings, please follow these instructions to enable it.
+
+1. Navigate to your AKS cluster in the [Azure Portal](https://portal.azure.com/).
+
+2. Click on **Diagnostic Settings** under **Monitoring** in the left navigation.
+
+3. Click on **Add diagnostic setting**.
+
+4. Enter a name, such as myAKSClusterLogs, then select the option to **Archive to a storage account**.
+
+5. Select a storage account of your choice.
+
+6. In the list of available logs, select the logs you wish to enable.
+    > Note: The incurred cost is subject to your storage account usage and Azure Storage Policy.
+
+7. When ready, select **Save** to enable collection of the selected logs.
+
+For more information on Diagnostics settings, visit [Create diagnostic settings to send platform logs and metics to different destinations](https://docs.microsoft.com/azure/azure-monitor/platform/diagnostic-settings).
+
+### Install Azure Service Operator
+
+Right click on your AKS cluster and click on **Install Azure Service Operator** to easily deploy the latest version of Azure Service Operator (ASO) on your AKS cluster and provision and connect applications to Azure resources within Kubernetes. When you select this option, you'll be prompted for a service principal for ASO to use when performing Azure resource operations. This service principal must have appropriate permissions (typically Contributor at suitable scope). Fill out the service principal details and click **Submit** to kick off the installation of Azure Service Operator.
```

Comment:
"and provision and connect applications to Azure resources" -> "to provision Azure resources"? Maybe it's just me but I'm not sure of the meaning of 'connecting applications' and I wonder if that's necessary.